src/SmartComponents/RunTaskModal/__tests__/TaskDescription.test.js

```diff
@@ -48,4 +48,19 @@ describe('TaskDescription', () => {
       })
     ).toHaveAttribute('href', '/api/tasks/v1/task/123/playbook');
   });
+
+  it('shows link in description', () => {
+    render(
+      <TaskDescription
+        description="Test description [link](https://www.google.com)"
+        slug="123"
+      />
+    );
+
+    expect(
+      screen.getByRole('link', {
+        name: /link/i,
+      })
+    ).toBeInTheDocument();
```

Comment:
I would add another assertion checking that `href` is set to `https://www.google.com`